src/Microsoft.VisualStudio.Editors/My Project/AssemblyInfo.vb

```diff
@@ -38,6 +38,13 @@ Imports System.Runtime.InteropServices
  "c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc09334" & _
  "4d5ad293")>
 
+<Assembly: InternalsVisibleTo("Microsoft.VisualStudio.PortableLibrary, PublicKey=" & _
```

Comment:
I also confirmed that the IVTs are up-to-date for the AppDesigners project: [VSO](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2Fwizard%2Fvbdesigner%2FAppDesigner%2FMy%20Project%2FAssemblyInfo.vb&version=GBmaster&_a=contents) and [GitHub](https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.AppDesigner/My%20Project/AssemblyInfo.vb)